javascript-local-storage/data.js

```diff
@@ -0,0 +1,10 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
You should have a conditional here to check if there was a value stored in localStorage, and only update the value of `todos` if there was a value

Reply:
https://user-images.githubusercontent.com/72685552/138536072-0835f97a-d9ca-4bbd-8a9d-b866dc5d348c.mov